src/jshint.js

```diff
@@ -3533,7 +3535,6 @@ var JSHINT = (function() {
           }
         }
       } else {
-        pastRest = spreadrest("rest");
         ident = identifier(context);
```

Comment:
You're right, this is pretty amazing!

Reply:
Hahah, I don't know if I said "amazing," but I'll take it!